skrub/_drop_column_if_null.py

```diff
@@ -8,8 +8,22 @@
 
 
 class DropColumnIfNull(SingleColumnTransformer):
```

Comment:
should we change the name to something like NullProportionThreshold ? :thinking:

Reply:
Maybe `DropColumnWithNullThreshold`? `DropIfNullAboveThreshold`?

Reply:
maybe!
DropIfTooManyNulls?

Reply:
+1 for DropIfTooManyNulls